### DISCUSSION TITLE
Feature: Lazy loading of poster images

### DISCUSSION THREAD

**Discussion by pushmatrix** [google/model-viewer#1879] [Q&A]
<model-viewer> already have lazy loading of 3D models in (which is awesome). It'd be nice to piggyback on that functionality and make the poster lazyload as well. Especially since posters don't have srcset support for multiple resolutions, it can take a lot of bandwidth if posters are big.

**Comment by cdata** [comment]
In general I agree and I think we will support this via the `poster` attribute.

That said, I'm interested to know: do you think you could address this case with something like `<img srcset="..." loading="lazy">`?

**Comment by pushmatrix** [comment]
Are you suggesting that it would be done through slots for poster?

**Comment by cdata** [comment]
@pushmatrix yah, something along the lines of:

```html
<model-viewer>
  <img srcset="..." loading="lazy" slot="poster">
</model-viewer>
```

Obligatory caveat: [`loading` attribute is only available in Chrome at this time.](https://caniuse.com/#feat=loading-lazy-attr)

**Comment by pushmatrix** [comment]
If we did that slot thing, would it break any of the css stuff that @elalish did for seamless poster transitions?

**Comment by elalish** [comment]
@pushmatrix No, if you use the same poster image and set `background-size: contain`, `background-repeat: no-repeat`, and `background-position: center`, it should work just fine.
